packages/legacy-scripting-runner/index.js

```diff
@@ -863,6 +863,19 @@ const legacyScriptingRunner = (Zap, zcli, input) => {
       templateContext
     );
 
+    const authFieldKeys = (_.get(app, 'authentication.fields') || []).map(
+      (f) => f.key
+    );
+    authFieldKeys.push('_zapier_account_id');
+
+    // In CLI, it's bundle.inputData instead of bundle.authData has yet-to-save
```

Comment:
```suggestion
    // In CLI, bundle.inputData instead of bundle.authData has yet-to-save
```

Reply:
I actually wanted to keep the "it's". Fixed the grammar in 30cc5ef.